.github/workflows/test.yml

```diff
@@ -132,7 +132,7 @@ jobs:
           os: ubuntu-20.04
           arch: x86_64
           action: test
-          flags: --config=clang --define=crypto=system
+          flags: --config=clang --define=crypto=system --test_timeout=600
```

Comment:
From https://github.com/dio/proxy-wasm-cpp-host/actions/runs/3179609270/jobs/5182308733#step:9:83 it takes about 382.0s

```
//test:wasm_vm_test                                                      PASSED in 382.0s
```